dataprofiler/profilers/float_column_profile.py

```diff
@@ -148,6 +139,44 @@ def profile(self):
         
         return profile
 
+    @property
+    def precision(self):
+        """
+        Property reporting statistics on the significant figures of each
+        element in the data.
+        :return
+        :rtype dict
```

Comment:
For return and rtype you need to close with a colon as well (":return:" and ":rtype:") or else the autodocumentation will fail. You should also give a description of what youre returning when you are returning something